src/client/datascience/jupyter/jupyterNotebook.ts

```diff
@@ -285,9 +285,13 @@ export class JupyterNotebookBase implements INotebook {
             }
 
             // Run any startup commands that we specified. Support the old form too
-            const setting = settings.runStartupCommands || settings.runMagicCommands;
+            let setting = settings.runStartupCommands || settings.runMagicCommands;
+
+            // Convert to string in case we get an array of startup commands.
+            if (Array.isArray(setting)) setting = setting.join(`\n`);
```

Comment:
Please use braces with if statements 😊 This will allow the [Build (Hygiene) task](https://dev.azure.com/ms/vscode-python/_build/results?buildId=93454&view=logs&j=b5d5f14c-0ff5-54d6-9f58-3b18f7065b47&t=2c1505b6-33f0-58b7-6dc3-18b694724987&l=75) to pass.

```suggestion
            if (Array.isArray(setting)) {
                    setting = setting.join(`\n`);
            } 
```